src/main/java/uk/gov/hmcts/reform/jobscheduler/model/Trigger.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.jobscheduler.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.ZonedDateTime;
+
+public class Trigger {
+
+    public final Frequency frequency;
+    public final Integer interval;
+    public final ZonedDateTime startDateTime;
+
+    public Trigger(
+        @JsonProperty("frequency") Frequency frequency,
+        @JsonProperty("interval") Integer interval,
+        @JsonProperty("start_date_time") ZonedDateTime startDateTime
+    ) {
+        this.frequency = frequency;
+        this.interval = interval;
+        this.startDateTime = startDateTime;
+    }
+
+    public enum Frequency {
+        second,
+        minute,
+        hour,
+        day,
```

Comment:
I wonder if we need weekday, as well. Also, not sure about the casing.

Reply:
names are based on Logic Apps api

Reply:
Maybe `SECOND("second")` or something like that, so that the mapping is more explicit. What do you think?

Reply:
the string value is clear to me the way it is tbh

Reply:
What I meant is that at some point someone may change the names here to match the Java naming convention and that will break things.

Reply:
fair enough, will change